safenode/src/client/api.rs

```diff
@@ -167,16 +167,19 @@ impl Client {
     pub(super) async fn store_chunk(&self, chunk: Chunk) -> Result<()> {
         info!("Store chunk: {:?}", chunk.address());
         let request = Request::Cmd(Cmd::StoreChunk(chunk));
+        // TODO: consider using `expect_closest_majority_ok` as within the dbc spend flow
```

Comment:
Can we put this in an issue please. Otherwise this is good to go.

Reply:
issue https://github.com/maidsafe/safe_network/issues/247  crated for this